### DISCUSSION TITLE
Youtube Music in Chrome?

### DISCUSSION THREAD

**Discussion by Longshot9** [web-scrobbler/web-scrobbler#3564] [General]
Any tips for getting this extension to work with Youtube Music in Chrome?  Says the website is not supported when I try it in Chrome, but it works just fine in Edge.  Youtube Music is checked in the settings.  thx!

**Comment by dkordik** [comment]
I'm just a random passerby, but are you using "music.youtube.com", or some other URL? the rule seems to be set up to detect based on that URL, so if you're going in some other way, that might be why. if you're using some other URL, someone can probably update the matcher to treat that URL as YouTube Music as well

source: https://github.com/web-scrobbler/web-scrobbler/blob/v4.0.0/src/core/connectors.ts#L1139
